src/gui/LayoutMapper.h

```diff
@@ -56,13 +59,14 @@ class LayoutMapper
 	 * @param  settings  The Settings from which users settings are obtained
 	 */
 
-	
-	void configureFromSettings(int numPages, Settings* settings);
-	
-	
+
+	void configureFromSettings(size_t numPages, Settings* settings);
+
+	[[deprecated ("use LayoutMapper::configureFromSettings")]]
```

Comment:
Since this is internal code, you can just remove the method if it doesn't break anything.